src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,363 @@
+                // the device is reset on the Raspberry PI.  I think the reset signal causes
+                // a disruption in the I2C connection.  Without shallowing this exception,
+                // Reset() would always throw the IOException.
+                if (ex.Message != "Error 121 performing I2C data transfer.")
+                {
+                    throw;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Measure distance.
+        /// </summary>
+        /// <remarks>
+        /// Note: Do not call if running while in repetition mode.  It will block until
+        /// repetition finishes (forever if infinite).
+        /// </remarks>
+        /// <param name="withReceiverBiasCorrection">Faster without bias correction, but more prone to errors if condition changes.</param>
+        /// <returns>Distance as a length unit.</returns>
+        public Length MeasureDistance(bool withReceiverBiasCorrection = true)
+        {
+            WriteRegister(Register.ACQ_COMMAND, withReceiverBiasCorrection ? (byte)0x04 : (byte)0x03);
+
+            while (Status.HasFlag(SystemStatus.BusyFlag))
+            {
+                DelayHelper.DelayMicroseconds(4, allowThreadYield: true);
+            }
+
+            return LastDistance;
+        }
+
+        /// <summary>
+        /// Set the repetition mode to enable automatic measurement.
+        /// </summary>
+        /// <param name="measurementRepetition">Repetition mode, either Off, Repeat, or RepeatInfinitely.</param>
+        /// <param name="count">If Repeat, the number of times to repeat the measurement.</param>
+        /// <param name="delay">The delay between each measurements. Note the unit does not directly to hz, a value of 20 maps to about 100 hz.</param>
+        public void SetMeasurementRepetitionMode(MeasurementRepetition measurementRepetition, int count = -1, int delay = -1)
+        {
+            if (count < 2 || count > 254)
+            {
+                throw new ArgumentOutOfRangeException("Count must be between 2 and 254");
+            }
+
+            switch (measurementRepetition)
+            {
+                case MeasurementRepetition.Repeat:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, (byte)count);
+                    break;
+                case MeasurementRepetition.RepeatIndefinitely:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0xff);
+                    break;
+                case MeasurementRepetition.Off:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0x00);
+                    break;
+            }
+
+            if (delay != -1)
+            {
+                WriteRegister(Register.MEASURE_DELAY, (byte)delay);
+
+                // Set mode to use custom delay.
+                var currentAcqSettings = AcquisitionSettings;
+                currentAcqSettings |= AcquisitionSettings.UseDefaultDelay;
```

Comment:
this seems quite reverse to what you want: isn't UseDefaultDelay ignore `MEASURE_DELAY` setting?

Reply:
Good point!  Logic as written is correct, the flag name is mislabeled.  Documentaiton states that:

```
0: Use default delay for burst and free running mode.
1: Use delay from MEASURE_DELAY (0x45) for burst and free running mode
```
So `UseDefaultDelay` should be relabeled to `UseCustomDelay`.  I'm going to do that.